api/aether-top-level-openapi3.yaml

```diff
@@ -131,79 +131,311 @@ components:
       required:
         - target_id
       type: object
+    Failure:
+      properties:
+        type:
+          description: transaction failure type
+          type: string
+          enum:
+            - UNKNOWN
+            - CANCELED
+            - NOT_FOUND
+            - ALREADY_EXISTS
+            - UNAUTHORIZED
+            - FORBIDDEN
+            - CONFLICT
+            - INVALID
+            - UNAVAILABLE
+            - NOT_SUPPORTED
+            - TIMEOUT
+            - INTERNAL
+        description:
+          type: string
+    Revision:
+      type: integer
+      format: int64
+    Synchronicity:
+      type: string
+      enum:
+        - ASYNCHRONOUS
+        - SYNCHRONOUS
+    Isolation:
+      type: string
+      enum:
+        - DEFAULT
+        - SERIALIZABLE
+    Strategy:
+      properties:
+        synchronicity:
+          description: indicates the transaction synchronicity level
+          $ref: '#/components/schemas/Synchronicity'
+        isolation:
+          description: indicates the transaction isolation level
+          $ref: '#/components/schemas/Isolation'
+    Index:
+      type: integer
+      format: int64
+    RollbackTransaction:
+      properties:
+        rollback_index:
+                description: the index of the transaction to roll back
+                $ref: '#/components/schemas/Index'
+    Deleted:
+      type: boolean
+    Path:
+      type: string
+    TypeOpts:
+      type: integer
+      format: int32
+    Bytes:
+      type: string
+      format: byte
+    ValueType:
+      description: the type for a value
+      type: string
+      enum:
+        - EMPTY
+        - STRING
+        - INT
+        - UINT
+        - BOOL
+        - DECIMAL
+        - FLOAT
+        - BYTES
+        - LEAFLIST_STRING
+        - LEAFLIST_INT
+        - LEAFLIST_UINT
+        - LEAFLIST_BOOL
+        - LEAFLIST_DECIMAL
+        - LEAFLIST_FLOAT
+        - LEAFLIST_BYTES
+    TypedValue:
+      description: value represented as a byte array
+      properties:
+        bytes:
+          description: the bytes array
+          $ref: '#/components/schemas/Bytes'
+        type:
+          description: the value type
+          $ref: '#/components/schemas/ValueType'
+        type_opts:
+          description: a set of type options
+          type: array
+          items:
+            $ref: '#/components/schemas/TypeOpts'
+    PathValue:
+      description: the state of a path/value in the configuration tree
+      properties:
+        path:
+          description: the path to change
+          $ref: '#/components/schemas/Path'
+        value:
+          description: the change value
+          $ref: '#/components/schemas/TypedValue'
+        deleted:
+          description: indicates whether this is a delete
+          $ref: '#/components/schemas/Deleted'
+    PathTarget:
+      type: object
+      properties:
+        target-name:
+          type: string
+        path-value:
```

Comment:
I think you can call this `path-values` instead

Reply:
I think on the line number 255 should be path-values

Reply:
oh - you're right

Reply:
Change path-value to path values at line 255